neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_direction(tree, soma):
+    '''Vector of trunk direction defined as
+       (initial tree point - soma center) of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
+    '''Angle between x-axis and vector defined by (initial tree point - soma center)
+       on the x-y half-plane.
+
+       Returns:
+           Angle in radians between -pi/2 and pi/2
+    '''
+    vector = trunk_direction(tree, soma)
+
+    if np.linalg.norm(vector) != 0:
```

Comment:
It is better than 0, but you should check if numpy provides similar epsilon numbers that depend on the type of the expression being evaluated (`np.linalg.norm(v)`)